Errors occurs while executes the command "npm run release"

When executes the command "npm run release", the following error occurs:

```
> PM&gt; npm run release
> npm : npm ERR! path C:\Users\leiyang\source\repos\SignalRChat\package.json
> At line:1 char:1
> + npm run release
> + ~~~~~~~~~~~~~~~
>     + CategoryInfo          : NotSpecified: (npm ERR! path C...at\package.json:String) [], RemoteException
>     + FullyQualifiedErrorId : NativeCommandError
>  
> npm ERR!
>  
> code
>  ENOENT
> 
> npm
>  ERR! errno -4058
> 
> npm
>  
> ERR!
>  
> syscall
>  open
> 
> npm
>  ERR! 
> enoent
>  ENOENT: no such file or directory, open 'C:\Users\leiyang\source\repos\SignalRChat\package.json'
> 
> npm
>  ERR! enoent This is related to npm not being able to find a file.
> npm ERR! 
> enoent
> npm
>  
> ERR!
>  A complete log of this run can be found in:
> npm ERR!
>      C:\Users\leiyang\AppData\Roaming\npm-cache\_logs\2018-11-30T07_53_48_451Z-debug.log
> 
> PM&gt; 
> 

```
---

Screenshot as below:


![error-signal](https://user-images.githubusercontent.com/25738099/49276602-ec933e80-f4b9-11e8-99ab-f276adb043d7.JPG)

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 47557f89-3a14-1139-0074-38d931b35faf
* Version Independent ID: 1cee43b4-b672-e480-3e26-72e08a6c2d77
* Content: [Use ASP.NET Core SignalR with TypeScript and Webpack](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr-typescript-webpack?view=aspnetcore-2.1&tabs=visual-studio#create-the-aspnet-core-web-app)
* Content Source: [aspnetcore/tutorials/signalr-typescript-webpack.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/signalr-typescript-webpack.md)
* Product: **aspnet-core**
* GitHub Login: @ssougnez
* Microsoft Alias: **scaddie**

--- a/aspnetcore/tutorials/signalr-typescript-webpack.md
+++ b/aspnetcore/tutorials/signalr-typescript-webpack.md
@@ -33,7 +33,7 @@ In this tutorial, you learn how to:
 
 Configure Visual Studio to look for npm in the *PATH* environment variable. By default, Visual Studio uses the version of npm found in its installation directory. Follow these instructions in Visual Studio:
 
-1. Navigate to **Tools** > **Options** > **Projects and solutions** > **Web Package Management** > **External Web Tools**.
+1. Navigate to **Tools** > **Options** > **Projects and Solutions** > **Web Package Management** > **External Web Tools**.
 1. Select the *$(PATH)* entry from the list. Click the up arrow to move the entry to the second position in the list.
 
     ![Visual Studio Configuration](signalr-typescript-webpack/_static/signalr-configure-path-visual-studio.png)
@@ -207,7 +207,7 @@ Confirm that the app works with the following steps.
 
 # [Visual Studio](#tab/visual-studio)
 
-1. Run Webpack in *release* mode. Using the **Package Manager Console** window, execute the following command in the project root:
+1. Run Webpack in *release* mode. Using the **Package Manager Console** window, execute the following command in the project root. If you are not in the project root, enter `cd SignalRWebPack` before entering the command.
 
     [!INCLUDE [npm-run-release](../includes/signalr-typescript-webpack/npm-run-release.md)]
 

